_templates/vue/component/prompt.js

```diff
@@ -8,7 +8,7 @@ module.exports = [
     {
         type: 'input',
         name: 'name',
-        message: 'What is the name of your vue component?',
+        message: 'What is the name of your vue component? (PascalCase)',
```

Comment:
@kingy04 is this clear enough or do you think it needs more?

Reply:
This is fine thanks